vtds_cluster_kvm/private/cluster.py

```diff
@@ -321,8 +327,28 @@ def __add_host_blade_net(self):
                 }
             }
             node_class['network_interfaces'][netname] = host_blade_interface
-        self.config['networks'] = networks
-        self.config['node_classes'] = node_classes
+        config['networks'] = networks
+        config['node_classes'] = node_classes
+
```

Comment:
REVIEWERS: there was a multiple source of truth issue in the configuration where the list of connected blade types and instances were implied by the AF_INET address family configuration of a network, but also needed to be explicitly stated at the top level of the network configuration. By computing the explicit configuration at the top level and adding it here, I make it easier for the deployment script to work while making sure that there is only one source of truth for the connected blades on a network.